src/mbgl/map/source.cpp

```diff
@@ -188,7 +190,7 @@ void Source::render(Painter &painter, const StyleLayer &layer_desc) {
     gl::group group(std::string { "layer: " } + layer_desc.id);
     for (const auto& pair : tiles) {
         Tile &tile = *pair.second;
-        if (tile.data && tile.data->state == TileData::State::parsed) {
+        if (tile.data && tile.data->ready()) {
```

Comment:
Can we change this into something that indicates that state will be changed, like `setReady()`?